packages/calcite-components/src/components/list/list.scss

```diff
@@ -7,12 +7,13 @@
  */
 
 :host {
-  @apply block;
+  @apply flex;
```

Comment:
This would be a visual breaking change since we would be changing the default display value of the component from `block` to `flex`. This may be acceptable as the results are probably minor but we would need to note it regardless. @ashetland @jcfranco would we want to wait for 4.0 to do this?

In the meantime, we could add a `min-height` to the internal `container` of the component when there are no items inside it and leave the host as `block` display.

Reply:
A `min-height` might make sense in the long run. I'm thinking forward to a time when we have skeletons (#5162).